core/server/services/webhooks/listen.js

```diff
@@ -1,6 +1,6 @@
 var _ = require('lodash'),
-    common = require('../lib/common'),
-    api = require('../api'),
+    common = require('../../lib/common'),
+    api = require('../../api'),
```

Comment:
> Is this a valid module anymore? Should it have /v0.1 or /v2 appended?
It automatically defaults to `v0.1` right now, was refactored this way since already lot of places use this direct require without explicit version
> should you just drop this module entirely and use the services trigger mechanism?
I thought about it initially and did the changes, but that led to changes needed in v0.1 tests as well since `listen` is already used there. Since we were trying to do min changes for existing v0.1, I didn't update `listen` function for now and it uses the old logic 😬